<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHintDialog</class>
 <widget class="QDialog" name="UserHintDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Title</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="textEditHint">
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>68</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="checkShowHint">
     <property name="text">
      <string>Show this hint again</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton2">
     <property name="text">
      <string>Button2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton1">
     <property name="text">
      <string>Button1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>UserHintDialog</receiver>
   <slot>button2clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>UserHintDialog</receiver>
   <slot>button1clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button1clicked()</slot>
  <slot>button2clicked()</slot>
 </slots>
</ui>
